plugins/gatsby-theme-iterative-docs/src/components/Documentation/Markdown/Main/styles.module.css

```diff
@@ -277,6 +277,7 @@
   .yt-embed-wrapper {
     position: relative;
     display: flex;
+    margin-block: 0 16px;
```

Comment:
For consistency, should we use `margin` instead of `margin-block` and just give the wrapper a `margin-bottom` of 16px? Almost all of our md elements just have `margin-bottom`.

Reply:
Sure.